src/engine/runtime.js

```diff
@@ -509,7 +682,30 @@ Runtime.prototype._updateScriptGlows = function () {
             finalScriptGlows.push(currentFrameGlow);
         }
     }
+    for (var m = 0; m < this._blockGlowsPreviousFrame.length; m++) {
+        var previousBlockGlow = this._blockGlowsPreviousFrame[m];
+        if (requestedBlockGlowsThisFrame.indexOf(previousBlockGlow) < 0) {
+            // Glow turned off.
+            try {
+                this.glowBlock(previousBlockGlow, false);
```

Comment:
Is there a reasonable way to predict when this will throw and instead avoid the call in those situations? This type of "catch and suppress" situation has hidden bugs in Scratch 2.0 frequently enough that I get nervous when I see an empty `catch` clause...